2019/12/09 14:20:21 [error] 8#8: *4 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:20:22 [error] 8#8: *4 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2019/12/09 14:20:39 [error] 8#8: *9 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:20:47 [error] 8#8: *9 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /app_dev.php HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:35:21 [error] 8#8: *14 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:50:30 [error] 8#8: *18 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:51:32 [error] 8#8: *21 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/09 14:51:33 [error] 8#8: *21 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 11:51:23 [error] 9#9: *52 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot redeclare App\Controller\UserController::base() in /var/www/symfony/src/Controller/UserController.php on line 38PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:436PHP message: PHP   6. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/config/ConfigCache.php:60PHP message: PHP   7. Symfony\Component\Config\Resource\SelfCheckingResourceChecker->isFresh() /var/www/symfony/vendor/symfony/config/ResourceCheckerConfigCache.php:99PHP message: PHP   8. Symfony\Component\Config\Resource\ReflectionClassResource->isFresh() /var/www/symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php:34PHP message: PHP   9. Symfony\Component\Config\Resource\ReflectionClassResource->computeHash() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:54PHP message: PHP  10. ReflectionClass->__construct() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  11. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  12. Symfony\Component\ErrorHandler\DebugClassLoader->loadClass() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /default HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 11:51:25 [error] 9#9: *51 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot redeclare App\Controller\UserController::base() in /var/www/symfony/src/Controller/UserController.php on line 38PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:436PHP message: PHP   6. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/config/ConfigCache.php:60PHP message: PHP   7. Symfony\Component\Config\Resource\SelfCheckingResourceChecker->isFresh() /var/www/symfony/vendor/symfony/config/ResourceCheckerConfigCache.php:99PHP message: PHP   8. Symfony\Component\Config\Resource\ReflectionClassResource->isFresh() /var/www/symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php:34PHP message: PHP   9. Symfony\Component\Config\Resource\ReflectionClassResource->computeHash() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:54PHP message: PHP  10. ReflectionClass->__construct() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  11. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  12. Symfony\Component\ErrorHandler\DebugClassLoader->loadClass() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107" while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /deleteUser HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 12:12:11 [error] 9#9: *82 FastCGI sent in stderr: "PHP message: 2019-12-16T12:12:11+00:00 [critical] Uncaught Exception: Cannot autowire service "App\Repository\UsersRepository": argument "$entityManager" of method "__construct()" references class "Doctrine\ORM\EntityManager" but no such service exists. Try changing the type-hint to "Doctrine\ORM\EntityManagerInterface" instead" while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /deleteUser HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 12:12:25 [error] 9#9: *82 FastCGI sent in stderr: "PHP message: 2019-12-16T12:12:25+00:00 [critical] Uncaught Exception: Cannot autowire service "App\Repository\UsersRepository": argument "$entityManager" of method "__construct()" references class "Doctrine\ORM\EntityManager" but no such service exists. Try changing the type-hint to "Doctrine\ORM\EntityManagerInterface" instead" while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /userBase HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 12:12:49 [error] 9#9: *82 FastCGI sent in stderr: "PHP message: 2019-12-16T12:12:49+00:00 [critical] Uncaught Exception: Cannot autowire service "App\Repository\UsersRepository": argument "$entityManager" of method "__construct()" references class "Doctrine\ORM\EntityManager" but no such service exists. Try changing the type-hint to "Doctrine\ORM\EntityManagerInterface" instead" while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /userBase HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 12:51:48 [crit] 9#9: *116 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /newUser HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:49:40 [crit] 9#9: *151 open() "/var/tmp/nginx/fastcgi/2/00/0000000002" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:53:59 [crit] 9#9: *163 open() "/var/tmp/nginx/fastcgi/3/00/0000000003" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:54:11 [error] 9#9: *167 FastCGI sent in stderr: "PHP message: 2019-12-16T13:54:11+00:00 [critical] Uncaught Error: syntax error, unexpected '$query' (T_VARIABLE)" while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:55:24 [crit] 9#9: *173 open() "/var/tmp/nginx/fastcgi/4/00/0000000004" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:56:23 [crit] 9#9: *176 open() "/var/tmp/nginx/fastcgi/5/00/0000000005" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:56:58 [crit] 9#9: *178 open() "/var/tmp/nginx/fastcgi/6/00/0000000006" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:58:36 [crit] 9#9: *184 open() "/var/tmp/nginx/fastcgi/7/00/0000000007" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 13:59:47 [crit] 9#9: *192 open() "/var/tmp/nginx/fastcgi/8/00/0000000008" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:00:29 [error] 9#9: *198 FastCGI sent in stderr: "PHP message: 2019-12-16T14:00:29+00:00 [critical] Uncaught Error: syntax error, unexpected '$query' (T_VARIABLE)" while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:02:03 [crit] 9#9: *202 open() "/var/tmp/nginx/fastcgi/9/00/0000000009" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:02:25 [crit] 9#9: *206 open() "/var/tmp/nginx/fastcgi/0/01/0000000010" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:02:53 [crit] 9#9: *209 open() "/var/tmp/nginx/fastcgi/1/01/0000000011" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:03:07 [crit] 9#9: *211 open() "/var/tmp/nginx/fastcgi/2/01/0000000012" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:08:25 [crit] 9#9: *231 open() "/var/tmp/nginx/fastcgi/3/01/0000000013" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:10:15 [crit] 9#9: *241 open() "/var/tmp/nginx/fastcgi/4/01/0000000014" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:10:41 [crit] 9#9: *244 open() "/var/tmp/nginx/fastcgi/5/01/0000000015" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:10:55 [crit] 8#8: *247 open() "/var/tmp/nginx/fastcgi/6/01/0000000016" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:11:42 [crit] 9#9: *250 open() "/var/tmp/nginx/fastcgi/7/01/0000000017" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:12:19 [crit] 9#9: *253 open() "/var/tmp/nginx/fastcgi/8/01/0000000018" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:12:45 [crit] 9#9: *256 open() "/var/tmp/nginx/fastcgi/9/01/0000000019" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:15:17 [crit] 9#9: *258 open() "/var/tmp/nginx/fastcgi/0/02/0000000020" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:15:42 [crit] 9#9: *262 open() "/var/tmp/nginx/fastcgi/1/02/0000000021" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:16:29 [crit] 9#9: *265 open() "/var/tmp/nginx/fastcgi/2/02/0000000022" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:17:07 [crit] 9#9: *267 open() "/var/tmp/nginx/fastcgi/3/02/0000000023" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:17:23 [crit] 8#8: *271 open() "/var/tmp/nginx/fastcgi/4/02/0000000024" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:22:10 [crit] 9#9: *274 open() "/var/tmp/nginx/fastcgi/5/02/0000000025" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/16 14:22:34 [crit] 9#9: *276 open() "/var/tmp/nginx/fastcgi/6/02/0000000026" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET /fetchUsersContacts HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2019/12/17 11:31:39 [crit] 6#6: *8 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /newUser HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/17 11:37:35 [error] 6#6: *14 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Declaration of App\Repository\UsersRepository::newUser(App\Entity\Users $user) must be compatible with App\RepositoryInterfaces\UserInterface::newUser() in /var/www/symfony/src/Repository/UsersRepository.php on line 18PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. App\Kernel->buildContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:544PHP message: PHP   6. App\Kernel->registerContainerConfiguration() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:635PHP message: PHP   7. Symfony\Component\Config\Loader\DelegatingLoader->load() /var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:84PHP message: PHP   8. Symfony\Component\DependencyInjection\Loader\ClosureLoader->load() /var/www/symfony/vendor/symfony/config/Loader/DelegatingLoader.php:40PHP message: PHP   9. App\Kernel->Symfony\Bundle\FrameworkBundle\Kernel\{closure:/var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:59-84}() /var/www/symfony/vendor/symfony/dependency-injection/Loader/ClosureLoader.php:38PHP message: PHP  10. App\Kernel->configureContainer() /var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:81PHP message: PHP  11. Symfony\Component\Config\Loader\DelegatingLoader->load() /var/www/symfony/src/Kernel.php:42PHP message: PHP  12. Symfony\Component\DependencyInjection\Loader\GlobFileLoader->load() /var/www/symfony/vendor/symfony/config/Loader/DelegatingLoader.php:40PHP message: PHP  13. Symfony\Component\DependencyInjection\Loader\GlobFileLoader->import() /var/www/symfony/vendor/symfony/dependency-injection/L
2019/12/17 11:37:35 [error] 6#6: *14 upstream sent too big header while reading response header from upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /user/newUser HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/17 11:37:53 [error] 6#6: *17 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Declaration of App\Repository\UsersRepository::newUser(App\Entity\Users $user) must be compatible with App\RepositoryInterfaces\UserInterface::newUser() in /var/www/symfony/src/Repository/UsersRepository.php on line 18PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. App\Kernel->buildContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:544PHP message: PHP   6. App\Kernel->registerContainerConfiguration() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:635PHP message: PHP   7. Symfony\Component\Config\Loader\DelegatingLoader->load() /var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:84PHP message: PHP   8. Symfony\Component\DependencyInjection\Loader\ClosureLoader->load() /var/www/symfony/vendor/symfony/config/Loader/DelegatingLoader.php:40PHP message: PHP   9. App\Kernel->Symfony\Bundle\FrameworkBundle\Kernel\{closure:/var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:59-84}() /var/www/symfony/vendor/symfony/dependency-injection/Loader/ClosureLoader.php:38PHP message: PHP  10. App\Kernel->configureContainer() /var/www/symfony/vendor/symfony/framework-bundle/Kernel/MicroKernelTrait.php:81PHP message: PHP  11. Symfony\Component\Config\Loader\DelegatingLoader->load() /var/www/symfony/src/Kernel.php:42PHP message: PHP  12. Symfony\Component\DependencyInjection\Loader\GlobFileLoader->load() /var/www/symfony/vendor/symfony/config/Loader/DelegatingLoader.php:40PHP message: PHP  13. Symfony\Component\DependencyInjection\Loader\GlobFileLoader->import() /var/www/symfony/vendor/symfony/dependency-injection/L
2019/12/17 11:37:53 [error] 6#6: *17 upstream sent too big header while reading response header from upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /user/newUser HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/17 11:54:30 [error] 6#6: *67 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Declaration of App\Repository\UsersRepository::newUser(App\Entity\Users $user) must be compatible with App\RepositoryInterfaces\UserInterface::newUser() in /var/www/symfony/src/Repository/UsersRepository.php on line 18PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:436PHP message: PHP   6. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/config/ConfigCache.php:60PHP message: PHP   7. Symfony\Component\Config\Resource\SelfCheckingResourceChecker->isFresh() /var/www/symfony/vendor/symfony/config/ResourceCheckerConfigCache.php:99PHP message: PHP   8. Symfony\Component\Config\Resource\ReflectionClassResource->isFresh() /var/www/symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php:34PHP message: PHP   9. Symfony\Component\Config\Resource\ReflectionClassResource->computeHash() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:54PHP message: PHP  10. ReflectionClass->__construct() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  11. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  12. Symfony\Component\ErrorHandler\DebugClassLoader->loadClass() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  13. include() /var/www/symfony/vendor/symfony/error-handler/DebugClassLoader.php:335" while reading response header from upstream, client: 172.24.0.
2019/12/17 11:54:37 [error] 6#6: *67 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Declaration of App\Repository\UsersRepository::newUser(App\Entity\Users $user) must be compatible with App\RepositoryInterfaces\UserInterface::newUser() in /var/www/symfony/src/Repository/UsersRepository.php on line 18PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:186PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:126PHP message: PHP   5. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:436PHP message: PHP   6. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/config/ConfigCache.php:60PHP message: PHP   7. Symfony\Component\Config\Resource\SelfCheckingResourceChecker->isFresh() /var/www/symfony/vendor/symfony/config/ResourceCheckerConfigCache.php:99PHP message: PHP   8. Symfony\Component\Config\Resource\ReflectionClassResource->isFresh() /var/www/symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php:34PHP message: PHP   9. Symfony\Component\Config\Resource\ReflectionClassResource->computeHash() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:54PHP message: PHP  10. ReflectionClass->__construct() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  11. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  12. Symfony\Component\ErrorHandler\DebugClassLoader->loadClass() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:107PHP message: PHP  13. include() /var/www/symfony/vendor/symfony/error-handler/DebugClassLoader.php:332" while reading response header from upstream, client: 172.24.0.
2019/12/17 12:03:44 [crit] 6#6: *84 open() "/var/tmp/nginx/fastcgi/2/00/0000000002" failed (13: Permission denied) while reading upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /user/newUserTemplate HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/17 12:04:54 [crit] 6#6: *87 open() "/var/tmp/nginx/fastcgi/3/00/0000000003" failed (13: Permission denied) while reading upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /user/newUserTemplate HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/19 12:59:48 [error] 12#12: *87 FastCGI sent in stderr: "PHP message: 2019-12-19T12:59:48+00:00 [critical] Uncaught Error: syntax error, unexpected '}', expecting ';'" while reading upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /userBase HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
2019/12/19 14:11:30 [crit] 12#12: *175 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.24.0.1, server: symfony.localhost, request: "GET /userBase HTTP/1.1", upstream: "fastcgi://172.24.0.2:9001", host: "symfony.localhost"
